resolwe_bio/processes/clustering/hierarchical_clustering.yml

```diff
@@ -94,32 +94,66 @@
       type: basic:string
       required: false
     - name: distance
-      label: Distance measure
+      label: Distance metric
       type: basic:string
-      default: spearman
+      default: pearson
       choices:
-        - label: spearman
-          value: spearman
-        - label: pearson
-          value: pearson
-        - label: euclidean
+        - label: Euclidean
           value: euclidean
+        - label: Pearson
+          value: pearson
+        - label: Spearman
+          value: spearman
     - name: linkage
-      label: Clustering linkage
+      label: Linkage method
       type: basic:string
       default: average
       choices:
-        - label: average
-          value: average
         - label: single
           value: single
+        - label: average
+          value: average
         - label: complete
           value: complete
     - name: log2
```

Comment:
This fix requires changes also in other places where the process is called. Other visualizations currently also use `log2` as the name of input parameter. Therefore, it seems best to keep this as it is for now.